nixos/modules/services/x11/urxvtd.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+# maintainer: siddharthist
+
+with lib;
+
+let
+  cfg = config.services.urxvtd;
+in {
+
+  options.services.urxvtd.enable = mkOption {
+    type = types.bool;
+    default = false;
+    example = true;
+    description = ''
+      Enable urxvtd, the urxvt terminal daemon. To use urxvtd, run
+      export RXVT_SOCKET=/run/user/your-uid/urxvtd-socket and then
```

Comment:
@siddharthist forgot, what i said:

this is little bit more suitable for copy-n-paste

```bash
export RXVT_SOCKET=/run/user/$(id -u)/urxvtd-socket
```